core/pom.xml

```diff
@@ -481,6 +481,29 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>sparkr-docs</id>
```

Comment:
Why is this in `core` though? It doesn't look like the right place.
Shouldn't we have an `R/pom.xml` that handles this?